foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,44 +1,84 @@
 ### Introduction
 
-We're going to make an implementation of grade-school classic "rock paper scissors". If you don't know what that is check the [Wikipedia article](https://en.wikipedia.org/wiki/Rock%E2%80%93paper%E2%80%93scissors) or [this](https://www.wikihow.com/Play-Rock,-Paper,-Scissors) detailed step-by-step. For the moment we're just going to play the game from the browser console, but **we will revisit this project in a later lesson and add a Graphical User Interface with buttons and text,** so don't forget to keep the code on GitHub! You might notice some 'Live Preview' links in the student solutions that have a GUI - this is coming in a later lesson. When you get there don't forget to come back and add your link!
+For this project, you will create the game **Rock Paper Scissors**. This game will be played entirely on the console. To learn more, or if you're not familiar with the game, read this [Wikipedia article](https://en.wikipedia.org/wiki/Rock%E2%80%93paper%E2%80%93scissors) or this [detailed step-by-step guide.](https://www.wikihow.com/Play-Rock,-Paper,-Scissors)
 
-### Quick exercises before starting
+<div class="lesson-note">
+Some of the student solutions below contain buttons, text, and other elements. These elements are part of what is called a graphical user interface (GUI) You'll create the GUI in a later lesson. In the meantime, remember to commit your code to GitHub.
+</div>
+
+### Quick exercise before starting
+
+1. Identify three ways to include JavaScript in a webpage.
+1. Check if the webpage includes JavaScript:
+   - Write `console.log("Hello World")` in JavaScript.
+   - Check if "Hello World" is logged in the browser console once you open your webpage.
 
-1. Identify three ways to include JavaScript in a page.
-1. Test it out! Write `console.log("Hello World");` in JavaScript and check to see if it displays in the browser's console.
+### Problem solving approach
 
-Finally, this is your first JavaScript program built from scratch, so don't forget the previous lesson on problem solving. Plan your solution out before writing any code, and test each piece as you build to ensure it is working before moving on to the next!
+Remember the [previous wise words from the Problem Solving lesson](https://www.theodinproject.com/lessons/foundations-problem-solving) because this is your first JavaScript program built from scratch. Before you start this project, follow these steps for each step of the project:
```

Comment:
```suggestion
Remember the previous wise words from the [Problem Solving lesson](https://www.theodinproject.com/lessons/foundations-problem-solving) because this is your first JavaScript program built from scratch. Before you start this project, follow these steps for each step of the project:
```